dumb-init.c

```diff
@@ -67,6 +71,15 @@ void forward_signal(int signum) {
     }
 }
 
+void pause_signal(int signum) {
+    if (signum >= 1 && signum < MAXSIG) {
```

Comment:
Is this conditional necessary?  You parse arguments and check the bounds in`parse_pause_signum` function.  Is it ever possible for it to not be in this range?